pallets/automation-time/src/lib.rs

```diff
@@ -76,7 +79,8 @@ pub mod pallet {
 	use super::*;
 
 	pub type AccountOf<T> = <T as frame_system::Config>::AccountId;
-	pub type BalanceOf<T> = <<T as Config>::NativeTokenExchange as NativeTokenExchange<T>>::Balance;
+	pub type BalanceOf<T> =
```

Comment:
This was primary driver of this change. The `NativeTokenExchange::Balance` type is just different enough that passing balance from `automation-time` to any pallet except `balances` required (as best as I could figure out) an intermediate conversion to `u128`. Every other pallet I've seen uses the `Currency::Balance` type for `BalanceOf` and I think we should too.